spec/controllers/account_controller_spec.rb

```diff
@@ -0,0 +1,128 @@
+
+require "rails_helper"
+
+RSpec.describe AccountsController do
+  include SessionsHelper
+
+  let!(:account) {FactoryBot.create(:account)}
```

Comment:
nếu chị thay thành `let` thôi có dc ko

Reply:
dạ được. nhưng như vậy 1 lần gọi nó sẽ lại tạo thêm 1 account trong 1 block it ạ